core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -42,8 +60,25 @@
 import io.questdb.griffin.SqlInterruptorConfiguration;
 import io.questdb.log.Log;
 import io.questdb.mp.WorkerPoolConfiguration;
-import io.questdb.network.*;
-import io.questdb.std.*;
+import io.questdb.network.EpollFacade;
```

Comment:
Is it IDE setting to unwrap * into multiple includes? We better to have the same stile set up

Reply:
unwrap 2 lines into 1?